python/xdccq.py

```diff
@@ -1,8 +1,11 @@
 # coding=UTF-8
 
+from __future__ import print_function
+from builtins import str
+from builtins import range
```

Comment:
Hi,
As WeeChat doesn't support Python 2 any more, these 3 imports can safely be removed.
The `# coding=UTF-8` can also be removed (UTF-8 is default with Python 3).

Reply:
Hi,

script updated as requested.